internal/tokens/spl.go

```
@@ -0,0 +1,152 @@
+
+const (
+	tokenProgramID     = "TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA"
+	jsonRPCVersion     = "2.0"
+	discoveryRpcMethod = "getTokenAccountsByOwner"
+)
+
+type splDiscoveryService struct {
+	logger         *logrus.Logger
+	baseAddress    string
+	cmcService     *CMCService
+	oneInchService *oneInchService
```

golangci-lint (1.64.8)

field `oneInchService` is unused

(unused)